### DISCUSSION TITLE
Connection Refused Error

### DISCUSSION THREAD

**Discussion by yongra** [tl-its-umich-edu/my-learning-analytics#1336] [Q&A]
OK. I think I managed to get things set up. I got as far as it shows up on the course. When I click the external tool, I get the connection refused error.  In the LTI settings part of the env.hjson file, do I need to change the URLs to our school's Canvas URL? In the default file, they are all based on "canvas.instructure.com."

Thank you,

Ra

"/*  LTI 1.3 configuration": "*/",
"/*  The first key of LTI_CONFIG is the Canvas URL (production, beta, or test)": "*/",
"LTI_CONFIG": {
  "**https://canvas.instructure.com**": [
    {
      "/* requests without ID from this platform use this config by default": "*/",
      "default": true,
      "/* from Canvas: Developer Keys -> value from Details column": "*/",
      "client_id": "10000000000004",
      "/*Allowed hosts for the following 3 URLs: canvas.instructure.com, canvas.beta.instructure.com, canvas.test.instructure.com": "*/",
      "auth_login_url": "**http://canvas.instructure.com**/api/lti/authorize_redirect",
      "auth_token_url": "**http://canvas.instructure.com**/login/oauth2/token",
      "key_set_url": "**http://canvas.instructure.com**/api/lti/security/jwks",
      "/* this tool's private key ": "*/",
      "private_key_file": "/secrets/example_private.pem",
      "/* this tool's public key ": "*/",
      "public_key_file": "/secrets/example_public.pem",
      "/* Go to Canvas where this tool installed, click on the setting button (a gear icon, similar to '⚙️'), click 'Deployment Id', copy it, and paste here": "*/",
      "deployment_ids": [
        "9:0123456789abcdef0123456789abcdef01234567"
      ]
    }
  ]
},

**Comment by zqian** [comment]
@yongra FYI, I just checked with our prod deployment, those variables ("auth_login_url", "auth_token_url" and "key_set_url") still use the same values as shown above in the env_sample.hjson file.
